doc/os_api/endpoints/pure/contacts_endpoint.md

```diff
@@ -0,0 +1,265 @@
+Contacts endpoint (7)
+=============================
+* [Parameters explanation](#parameters-explanation)
+* [Pagination](#pagination)
+* [Usage examples](#usage-examples)
+  * [Get contacts count](#get-contacts-count)
+  * [Get contacts using offset and limit](#get-contacts-using-offset-and-limit)
+  * [Get a contact by ID](#get-a-contact-by-id)
+  * [Add a new contact](#add-a-new-contact)
+  * [Update a contact](#update-a-contact)
+  * [Delete a contact](#delete-a-contact)
+
+
+## Parameters explanation
+[Common parameters explanation](../../protocol_description/common_parameters_explanation.md)
+
+## Pagination 
+[Pagination](../../protocol_description/pagination.md)
+
+## Usage examples
+### Get contacts count
+**Note: For development/testing purposes only.**
+**Request Payload Structure**
+```
+{
+   "endpoint":7,
+   "method":1,
+   "uuid":1,
+   "body":{
+      "count":true
+   }
+}
+```
+**Response Payload Structure**
+```
+{
+   "body":{
+      "count":76
+   },
+   "endpoint":7,
+   "status":200,
+   "uuid":1
+}
+```
+
+### Get contacts using offset and limit
+See: [Pagination](../../protocol_description/pagination.md)
+**Request Payload Structure**
+```
+{
+   "endpoint":7,
+   "method":1,
+   "uuid":123,
+   "body":{
+      "offset":0,
+      "limit":2
+   }
+}
+```
+**Response Payload Structure**
+```
+{
+   "body":{
+      "entries":[
+         {
+            "address":"Czeczota 6, 02-600 Warsaw, Poland",
+            "altName":"Abraham",
+            "email":"Abraham@example.com",
+            "blocked":false,
+            "favourite":true,
+            "ice":true,
+            "id":1,
+            "numbers":[
+               "500675127"
```

Comment:
Please replace all phone numbers in these examples with ones that definitely do not belong to anyone (for eg. 123456789).